tgui/packages/tgui/interfaces/SkillPanel.js

```diff
@@ -21,7 +21,7 @@ export const SkillPanel = (props, context) => {
                   <p>XP to next level: [{skill.exp_prog} / {skill.exp_req}]</p>
                 )}
                 {skill.exp_req === 0 && (
-                  "XP to next level: [MAXXED]"
+                  "XP to nexts level: [MAXXED]"
```

Comment:
```suggestion
                  "XP to next's level: [MAXXED]"
```

Reply:
PR isn't quite ready for review yet (though it is close), as i'm just committing the changes I make as i go. This particular one was just to test if my tgui yarn dev server was running, as I'm in the process of reworking SkillPanel.js

I'll make a comment and remove the WIP tag when the code's ready to have a look taken at it